README.md

```diff
@@ -11,10 +11,15 @@ This crate exports all underlying platform types, functions, and constants under
 the crate root, so all items are accessible as `libc::foo`. The types and values
 of all the exported APIs match the platform that libc is compiled for.
 
+WinAPI bindings are not included in this crate. If you are looking for WinAPI
```

Comment:
Could you change the first `WinAPI` to `Windows API`? The shorthand is pretty common but we should mention the full name once.